package.json

```diff
@@ -25,13 +25,13 @@
   "dependencies": {
     "@bigcommerce/node-sass": "git://github.com/bigcommerce-labs/node-sass.git#v3.5.0",
     "autoprefixer": "^6.7.3",
-    "lodash": "^3.10.1",
+    "lodash": "4.17.19",
     "postcss": "^5.2.14"
   },
   "devDependencies": {
-    "code": "^4.0.0",
-    "eslint": "^3.15.0",
-    "lab": "^14.3.2",
+    "@hapi/code": "^8.0.2",
+    "@hapi/lab": "^23.0.0",
+    "eslint": "^7.6.0",
```

Comment:
I don't believe eslint 7 supports node 8. We need it bcoz we still have some code in production to support node 8. https://eslint.org/docs/user-guide/migrating-to-7.0.0#nodejs-8-is-no-longer-supported

Reply:
Let's move this to 6.x and hopefully everything will pass